docs/en/integrations/data-ingestion/etl-tools/dbt/index.md

```diff
@@ -820,6 +820,26 @@ This process is shown below:
 
 <img src={require('./images/dbt_06.png').default} class="image" alt="lightweight delete incremental" style={{width: '100%'}}/>
 
+### insert_overwrite mode (Experimental)
```

Comment:
Can you also point out the PR that introduced this feature since it is very interesting and important to understand how it works behind the hood